scripts/create_metadata

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -Eeuo pipefail
+
+# OUT_DIR is where the file gets written.
+# The main purpose of this is to make testing easier.
+OUT_DIR="${OUT_DIR:-.}"
+
+# Check if DEV_TAGS is unset, if not, set it to empty.
+if [ -z "${DEV_TAGS:-}" ]; then
+    DEV_TAGS=""
+fi
+
+# SPLIT splits a string by whitespace and removes any empty matches.
+SPLIT='split("[[:space:]]+"; "gn") | map(select(length > 0))'
+
+jq --null-input --sort-keys \
```

Comment:
I added `--sort-keys` so that we always get the keys in alphabetical order. This is handy for our assertion because we can use a simple string `diff` to check file content is as expected. (If keys were different order, the JSON objects are strictly still equal, it's just harder to check.)